Update sources

tresoldi opened the pull request:
This PR reviews the mapping for all sources (with the exception of `allenbai`, `beidasinitic`, and `bdpa`, which I did not touch), also adding a handful of new sounds to BIPA, such as "voiceless retroflex approximant", and regenerating all packages and the app.

| id            | valid    | total     | percent |
|------------|  -------|  -------|  ---------|
|allenbai   |   114  |    115   |        0.99|
|apics       |  177    |  177     |      1.00|
|bdpa      |    1329  |   1466  |        0.91|
|bdproto   |    745    |  794     |      0.94|
|beidasinitic | 145   |   145     |      1.00|
|chomsky     |  45    |   45        |    1.00|
|diachronica |  561  |    652      |     0.86|
|eurasian    |  1478  |   1562     |     0.95|
|jipa       |   937    |  957     |      0.98|
|lapsyd   |     793  |    795   |        1.00|
|multimedia |   137  |    138  |         0.99|
|nidaba    |    1927   |  1936   |       1.00|
|panphon  |     6272  |   6334  |        0.99|
|pbase     |    841    |  1068    |      0.79|
|phoible   |    3094  |   3182   |       0.97|
|powoco   |     371  |    378    |       0.98|
|ruhlen     |   533    |  701   |        0.76|
|sala       |   106    |  128    |       0.83|
|saphon   |     345  |    357  |         0.97|
|segbo     |    215   |   219    |       0.98|
|wiki        |  168    |  184       |    0.91|
|20          |             |          |  0.94 |

Commit:
Add multimedia, nidaba, and expand consonants

Commit:
Update remapped datasets with additional consonants

Commit:
Add pbase

Commit:
Add powoco

Commit:
Add ruhlen

Commit:
Add saphon

Commit:
Add segbo

Commit:
Add wiki; regenerate all dataset

cormacanderson commented on sources/nidaba/graphemes.tsv:
```diff
@@ -1,36 +1,25 @@
-<NA>	t͡ɬ̲ʷ	alveolar, labial:affricate:+voice +lateral	\t{t͡\textsubbar{\textbeltl}}\super w
-<NA>	t͡ɬ̲ʷː	alveolar, labial:affricate:+voice +lateral	\t{t͡\textsubbar{\textbeltl}}\super w\textlengthmark
-<NA>	t͡ɬ̲ː	alveolar:affricate:+voice +lateral	\t{t͡\textsubbar{\textbeltl}}\textlengthmark
+<NA>	d̠͡ɮ̠	dental:affricate:+voice +lateral	\t{\textsubbar{d}͡\textsubbar{\textlyoghlig}}
```
I would be inclined to just map these to dɮ etc. as you have done with the voiceless ones below

LinguList replied:
Makes sense, since this mapping is regular. Teh data was automatically produced by the author by combining features.

cormacanderson commented on sources/phoible/graphemes.tsv:
```diff
@@ -1328,8 +1329,8 @@ ts̪	ts̪	0:-:-:-:-:+:-:-:+:-:-:-:-:-:-:0:0:+:+:+:+:-:0:0:0:0:0:0:0:-:-:-:-:-:-:
 ts̻	ts̻	0:-:-:-:-:+:-:-:+:-:-:-:-:-:-:0:0:+:+:+:+:-:0:0:0:0:0:0:0:-:-:-:-:-:-:-:-	5	549E3ECF783015638FA48393A3D2F1FD
-tθ̠	ts͇	0:-:-:-:-:+:-:-:+:-:-:-:-:-:-:0:0:+:+:-:-:-:0:0:0:0:0:0:0:-:-:-:-:-:-:-:-	2	F107DE9F53FE9C9C6E30FEF515504BF5
-tθ̠ʰ	ts͇ʰ	0:-:-:-:-:+:-:-:+:-:-:-:-:-:-:0:0:+:+:-:-:-:0:0:0:0:0:0:0:-:-:+:-:-:-:-:-	1	6317AC4566C7464A377FF07157466BA3
+tθ͇	ts͇	0:-:-:-:-:+:-:-:+:-:-:-:-:-:-:0:0:+:+:-:-:-:0:0:0:0:0:0:0:-:-:-:-:-:-:-:-	2	F107DE9F53FE9C9C6E30FEF515504BF5
```
These should be the aliases.

cormacanderson commented on sources/phoible/graphemes.tsv:
```diff
@@ -1328,8 +1329,8 @@ ts̪	ts̪	0:-:-:-:-:+:-:-:+:-:-:-:-:-:-:0:0:+:+:+:+:-:0:0:0:0:0:0:0:-:-:-:-:-:-:
 ts̰	ts̰	0:-:-:-:-:+:-:-:+:-:-:-:-:-:-:0:0:+:+:-:+:-:0:0:0:0:0:0:0:+:-:-:+:-:-:-:-	6	9DCF9682DCFCE3E97E9BE1C7FCBC1612
 ts̺	ts̺	0:-:-:-:-:+:-:-:+:-:-:-:-:-:-:0:0:+:+:-:+:-:0:0:0:0:0:0:0:-:-:-:-:-:-:-:-	6	005BC670D0C514FD8B56234CB14F4FD5
 ts̻	ts̻	0:-:-:-:-:+:-:-:+:-:-:-:-:-:-:0:0:+:+:+:+:-:0:0:0:0:0:0:0:-:-:-:-:-:-:-:-	5	549E3ECF783015638FA48393A3D2F1FD
-tθ̠	ts͇	0:-:-:-:-:+:-:-:+:-:-:-:-:-:-:0:0:+:+:-:-:-:0:0:0:0:0:0:0:-:-:-:-:-:-:-:-	2	F107DE9F53FE9C9C6E30FEF515504BF5
```
These should be the default

cormacanderson commented on sources/phoible/graphemes.tsv:
```diff
@@ -2571,7 +2572,7 @@ l̻ˠ	ɫ̻	0:-:-:-:-:+:+:+:0:+:-:-:-:+:-:0:0:+:+:+:-:+:-:-:-:+:0:0:0:+:-:-:-:-:-
 ɯ̰ː̽	ɯ̰̽ː	0:-:+:-:+:-:+:+:0:+:-:-:-:-:-:0:0:-:0:0:0:+:0:-:-:0:0:-:-:+:-:-:+:0:-:-:0	1	D4E7E24EFC322C07907A57685F45E73B
 ɯ̽	ɯ̽	0:-:+:-:-:-:+:+:0:+:-:-:-:-:-:0:0:-:0:0:0:+:0:-:-:0:0:-:-:+:-:-:-:0:-:-:0	10	51027A5C6A1BDEAEA15ADE8085D4FEF9
 ɯ̽ː	ɯ̽ː	0:-:+:-:+:-:+:+:0:+:-:-:-:-:-:0:0:-:0:0:0:+:0:-:-:0:0:-:-:+:-:-:-:0:-:-:0	2	E427BE1B337CA8621A93DC9D811CBEC5
-ɯ͓	ɯ͓	0:-:+:-:-:-:+:+:+:+:-:-:-:-:-:0:0:-:0:0:0:+:+:-:-:+:+:-:-:+:-:-:-:0:-:-:0	2	CFE113C5FB098FA8AD3E61FAE8DDFAF0
```
Default. Alias below.

cormacanderson commented on sources/phoible/graphemes.tsv:
```diff
@@ -1653,7 +1654,7 @@ ṵ	ṵ	0:-:+:-:-:-:+:+:0:+:-:-:-:-:+:+:-:-:0:0:0:+:+:-:-:+:+:-:-:+:-:-:+:0:-:-:
 ṵː	ṵː	0:-:+:-:+:-:+:+:0:+:-:-:-:-:+:+:-:-:0:0:0:+:+:-:-:+:+:-:-:+:-:-:+:0:-:-:0	1	ED54C3F115E06055D3721C152E71B149
 ṵ̃	ṵ̃	0:-:+:-:-:-:+:+:0:+:-:-:+:-:+:+:-:-:0:0:0:+:+:-:-:+:+:-:-:+:-:-:+:0:-:-:0	4	75EEE8084A19F33CD18C68EA3405FD73
 u̽i	u̽i	0:-:+:-:-:-:+:+:0:+:-:-:-:-:+,-:+:-:-:0:0:0:+:+:-:-,+:-:+:-:-:+:-:-:-:0:-:-:0	1	F506A053B01C45A64BC782E64E7A8E0F
-u͓	u͓	0:-:+:-:-:-:+:+:+:+:-:-:-:-:+:+:-:-:0:0:0:+:+:-:-:+:+:-:-:+:-:-:-:0:-:-:0	1	614F55AF2945423229D3596FCD0CED32
```
Default. Alias below.

cormacanderson commented:
What is it that we are missing from LAPSyD? I thought everything was mapped there now.

LinguList commented:
@tresoldi, I cannot see this commit due to the file number. But tell me please quickly: did you add the new column which we call Symbols" to the tsv files in this run? If not, I suggest strongly to do this now, as it will otherwise again be a large PR. We later also modify pyclts to add the symbols as well in the pkg/-files.

LinguList commented:
@tresoldi, can you please answer my question on the Symbols? I'd like to know if this still needs to be advaanced or not? I mean, the CLTS commandd is already there, and routinely adds them. Just a quick reply please.

tresoldi commented:
@LinguList I kept looking here and can't understand which "Symbols column" you mean. I followed your example as much as I could, and it was the same workflow for the mappings we already merged (i.e., phoible, lapsyd, jipa).

LinguList commented:
@tresoldi, I wrote an email on this topic, and @cormacanderson and you 
both agreed that the feature was useful. Please pull the most recent 
pyclts code, and run the mapping one time, to see what I actually mean.

tresoldi commented:
I know, it is the email from 22 Nov 2020 20:10 and I agree it is a good idea. What I meant is that I cannot understand what I should have changed in the workflow.

For all sources, I ran `clts map` first, manually proceeded in correcting/improving the mappings, generated each package with `clts make_dataset`, and finally regenerated the app with `clts make_app`.

I did this with a `pyclts` from your `map-fix` branch, up to commit https://github.com/cldf-clts/pyclts/commit/5d58dccb484f440f9438bc68469b0b6ca08cf1f9 which is the most recent I can see there.

I can revert the changes due to `make_app`, and even due to all all `make_dataset`, if you prefer to get a simpler diff, after addressing the points raised by @cormacanderson first.

LinguList commented:
Tiago, I cannot view the PR since it is mega large. Don't you see that?
I ask you simply: do your files in our source folder HAVE the Systems
column, or do they NOT have it. Can you please just answer on that question?

tresoldi commented:
No, they don't have that column: https://github.com/cldf-clts/clts/blob/sources/sources/chomsky/graphemes.tsv

I ran `clts map` again from my setup and it is not adding it, unfortunately.

I will redo the PR only with the various `graphemes.tsv`, reverting all other files to the old versions.

cormacanderson commented:
I agree that the Symbols is a good idea.

However, if you just change the graphemes.tsv files in this PR, without making the necessary changes to pkg/transcriptionsystems/bipa/consonants.tsv  will some of the graphemes.tsv mappings not fail?

Does it make sense to first make sure that all of the mappings are fine and then add in the Symbols in a new PR straight afterwards?

LinguList commented:
Yep, so my proposal is, having clarified this: you two, @cormacanderson
and @tresoldi, figure out how to make sure all is correct, @tresoldi
merges this PR, and then we fix with the symbols, but this time:
step-by-step, not in the form of monster pull-requests, where too many
files are modified.

LinguList commented:
So to finish this, I'd suggest to quickly address teh points made by @cormacanderson and then merge. We can later step by step do the conversion with the symbols for better visibility.

tresoldi commented:
Ok, I will address the points by @cormacanderson , use the code that was merged into `pyclts`, regenerate, and push.

Commit:
Update consonants

Commit:
Update sources with SYMBOL column

Commit:
Update package and app

tresoldi commented:
Ok, I have updated the sources, addressed the issues, regenerated, and will now merge. Note that I did not touch `allenbai`, `beidasinitic`, and `bdpa`.
